pkg/features/kube_features.go

```diff
@@ -1017,6 +1010,8 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	genericfeatures.OpenAPIEnums: {Default: true, PreRelease: featuregate.Beta},
 
+	genericfeatures.RetryGenerateName: {Default: true, PreRelease: featuregate.Beta},
```

Comment:
nit: alphabetically this should go a couple lines below

Reply:
Okay I will update this as suggested, Thanks!!